src/styles/_catalog.scss

```diff
@@ -13,10 +13,13 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
There should be four columns **starting** at 1024px
There should be three columns **starting** at 768px
There should be two columns **starting** at 488px